backend/tests/test_partners.py

```diff
@@ -221,18 +220,22 @@ def test_partner_pagination(client, example_partners, access_token):
     assert res.status_code == 404
 
 
```

Comment:
We should add tests for the new API endpoints here. We're missing: 

- /invite
- /join
- /leave
- /remove_member
- /withdraw_invitation
- /role_change
- /invitations
- /stagedinvitations

Reply:
I've added tests for every endpoint except /invite. That one requires the mail server to be set up. Also, did not add tests for /invitations, and /staged_invitations as those are for the dev env, and I just wanted endpoints to view the current state of the db. We could remove them if you want.